### PR TITLE
Scott securian rec

### LINKED ISSUE #10: scott-securian checklist
- [ ] Create a branch
- [ ] Add a file
- [ ] Make a PR
- [ ] Merge

### PR DESCRIPTION
**Pull request by scott-securian**
Closes #10

### CODE REVIEW

File: scott-securian-rec.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# St. Paul, MN
+
+## Places to eat
+- Pizza Luce
+- Key's Cafe
```
Comment (karen-jakubowsky) [inline scott-securian-rec.md:5]: @scott-securian hi Scott.  adding a comment. do what you want.

Comment (karen-jakubowsky) [inline scott-securian-rec.md:5]: @scott-securian  can you add more?

Comment (karen-jakubowsky) [inline scott-securian-rec.md:5]: @scott-securian  can you add more?

Comment (karen-jakubowsky) [inline scott-securian-rec.md:5]: @scott-securian  can you add more?

Comment (karen-jakubowsky) [inline scott-securian-rec.md:5]: @scott-securian  can you add more?

Comment (scott-securian) [reply scott-securian-rec.md:5]: @karen-jakubowsky yep

**Review by karen-jakubowsky — APPROVED** [review]
Looks awesome!!!

**Pull request merged**